std-bits/table/src/main/java/org/enso/table/data/column/storage/type/TextType.java

```diff
@@ -15,6 +15,10 @@ public static TextType variableLengthWithLimit(long maxLength) {
   }
 
   public boolean fits(String string) {
+    if (string == null) {
```

Comment:
Is null a valid representation for an empty string? It seems like this should be filtered out earlier.

Reply:
It's not an empty string, it's a missing string and it is valid.